src/main/java/io/jenkins/plugins/zscaler/RunScanTask.java

```diff
@@ -159,7 +159,7 @@ private String runScan(TaskListener listener, String workspace) throws IOExcepti
       ZscannerSetup.cleanup(binaryLoc);
     }
 
-    if (StringUtils.isBlank(resp)) {
+    if (StringUtils.isEmpty(resp)) {
       // if result is null then err in above try block will be populated
```

Comment:
@jragula-zs this change is made because, if there are no iac files in a scan, then we were seeing below exception in lenkins console:

```
A JSONObject text must begin with '{' at 1 [character 2 line 1]
FATAL: A JSONObject text must begin with '{' at 1 [character 2 line 1]
org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:521)
	at org.json.JSONObject.<init>(JSONObject.java:225)
	at org.json.JSONObject.<init>(JSONObject.java:421)
	at io.jenkins.plugins.zscaler.ZscalerScan.validateAndFailBuild(ZscalerScan.java:187)
	at io.jenkins.plugins.zscaler.ZscalerScan.setUp(ZscalerScan.java:99)
	at jenkins.tasks.SimpleBuildWrapper.setUp(SimpleBuildWrapper.java:294)
	at hudson.model.Build$BuildExecution.doRun(Build.java:158)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:524)
	at hudson.model.Run.execute(Run.java:1899)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
	at hudson.model.ResourceController.execute(ResourceController.java:107)
	at hudson.model.Executor.run(Executor.java:449) 
```